tslint.json

```diff
@@ -57,7 +57,7 @@
       true,
       {
         "enforceBuildableLibDependency": true,
-        "allow": [],
+        "allow": ["@tmo/shared"],
```

Comment:
What's this enabling?

You might want to read up on tags -> https://nx.dev/latest/angular/structure/monorepo-tags

Reply:
reverted